sdk/client-core/core/src/main/java/io/clientcore/core/http/models/ServerSentEvent.java

```diff
@@ -13,7 +13,7 @@
  * <a href="https://html.spec.whatwg.org/multipage/server-sent-events.html#parsing-an-event-stream">here</a>.
  */
 public final class ServerSentEvent {
-    private Long id;
+    private String id;
```

Comment:
Just for my own understanding, what motivated this change? Are ids not always a long?

Reply:
yeah we just discussed this in the chat.
The spec https://html.spec.whatwg.org/multipage/server-sent-events.html doesn't specify a type and it so it isn't guaranteed to be a long.